Padre/padreEnvelopeProcessor.cpp

```diff
@@ -222,11 +223,18 @@ EnvelopeProcessor::ErrorCode EnvelopeProcessor::getTrackEnvelopeMinMax(TrackEnve
 	return eERRORCODE_OK;
 }
 
-void EnvelopeProcessor::writeLfoPoints(string &envState, double dStartTime, double dEndTime, double dValMin, double dValMax, LfoWaveParams &waveParams, double dPrecision, LfoWaveParams* freqModulator)
+void EnvelopeProcessor::writeLfoPoints(MediaItem_Take* take, string &envState, double dStartTime, double dEndTime, double dValMin, double dValMax, LfoWaveParams &waveParams, double dPrecision, LfoWaveParams* freqModulator)
 {
 	double dFreq, dDelay;
 	getFreqDelay(waveParams, dFreq, dDelay);
 
+	// account for take playrate, #1158
+	if (take)
+	{
+		double playrate = GetMediaItemTakeInfo_Value(take, "D_PLAYRATE");
+		dFreq = dFreq / playrate;
+	}
+

```

Comment:
This seems unrelated to the buffer change, and should probably be in a separate commit (doesn't have to be a new branch and PR, just a commit)?